app/routes/events/components/EventDetail/index.js

```diff
@@ -129,6 +149,17 @@ export default class EventDetail extends Component<Props> {
       ? () => unfollow(event.isUserFollowing.id, event.id)
       : () => follow(currentUser.id, event.id);
 
+    const fromToDate = (
```

Comment:
pls this one tooo

```jsx
<FromToTime from={event.startTime} to={event.endTime} />
```

Reply:
Really? For a one-time thing?:o

Reply:
It makes reading the component a lot smoother

Reply:
It should also be possible to reuse it in meetings.